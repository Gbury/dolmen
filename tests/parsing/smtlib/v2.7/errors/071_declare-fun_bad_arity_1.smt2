(set-logic ALL)
(declare-sort foo _)
