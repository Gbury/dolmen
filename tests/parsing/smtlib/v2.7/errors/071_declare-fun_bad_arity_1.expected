File "tests/parsing/smtlib/v2.7/errors/071_declare-fun_bad_arity_1.smt2", line 2, character 18-19:
2 | (declare-sort foo _)
                      ^
Error while parsing a sort declaration, read the reserved word '_',
      but expected a numeral for the arity of the sort being declared.
